src/probability/borelScript.sml

```diff
@@ -69,9 +69,10 @@ Definition Borel :
                          S IN {EMPTY; {NegInf}; {PosInf}; {NegInf; PosInf}}})
 End
 
-(* MATHEMATICAL DOUBLE-STRUCK CAPITAL B *)
+(* MATHEMATICAL DOUBLE-STRUCK CAPITAL B
```

Comment:
In order to get the manuals to build with new Unicode characters, you need to also adjust the `umap` file in `Manual/Tools`.  (Yes, the fact that we have two such maps is unfortunate.)

Reply:
I know `umap`, but now I think it’s not a very good choice to use these single-letter unicode chars (because each such  `umap` chars can only map to one LaTeX representation.).

Also the resulting terms are not pretty (too small) in Emacs. In the future I want to only keep the TeX notation for these constants: `borel`. `Borel` (and `Lebesgue`), but not map them to special Unicode chars.